app/components/buttons/EyeFilterMenu/EyeFilterMenuWithSlider/EyeFilterMenuWithSlider.jsx

```diff
@@ -40,7 +40,7 @@ const EyeFilterMenuWithSliderMenu = ({
   });
 
   const onChangeSliderCallback = useCallback(
-    debounce((value, limit) => {
+    () => debounce((value, limit) => {
```

Comment:
``lodash`` ``debounce`` returns a debounced version of its parameter. To call ``onChangeSliderCallback`` after this change will return the debounced function instead of executing it.

Reply:
Please see the review fix commit. The simplest way I could see to fix this without resorting to disabling the warning (as in the other issue Amass pointed out) was to switch to `useMemo`: the inline function now _produces_ the memoized, debounced callback (and correctly recreates it if `onChangeSlider` itself changes).

It seems to me though, looking at where `onChangeSlider` is ultimately defined (the `HistoryTab` component) it's being redefined at every component render there, thus causing unnecessary renders of the menu.

Anyways, hopefully this is correct. I also bumped up the delay a bit. 100ms seems way too small to debounce something that depends on fine use motion of the mouse.

Reply:
@matheusd That sounds as a good way to solve that, I would try wrapping `onChangeSliderValue` in `HistoryTab` with `useCallback` which should define the function only once.